torchao/experimental/ops/mps/test/test_quantizer.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional
+import copy
+import os
+import sys
+
+import torch
+import torchao_mps_ops
+import unittest
+
+torchao_root: Optional[str] = os.getenv("TORCHAO_ROOT")
+assert torchao_root is not None, "TORCHAO_ROOT is not set"
+
+sys.path.insert(0, torchao_root)
+from torchao.experimental.quant_api import IntxWeightOnlyLinearQuantizer
+
+
+def parameterized(test_cases):
```

Comment:
isnt there something equivalent within torchao?

Reply:
well, there is something in torch.testing._internal, but then I ran into dependency issues with 'expecttest'.

Reply:
we use this in other places: https://github.com/pytorch/ao/blob/c546c5c4ea5acf46067876470a5fb52f14f7e7fb/test/integration/test_integration.py#L647, not sure why there is a dependency issue

Reply:
done!